samples/colab/edge_detection.ipynb

```diff
@@ -66,7 +66,7 @@
         "outputId": "3ab6a4c6-46c2-45d2-9721-266b56d1d627"
       },
       "source": [
-        "!python -m pip install iree-compiler iree-runtime iree-tools-tf -f https://iree-org.github.io/iree/pip-release-links.html"
+        "!python -m pip install iree-compiler iree-runtime iree-tools-tf -f https://openxla.github.io/iree/pip-release-links.html"
```

Comment:
pip follows the redirect we set up, so users won't _need_ to update these flags, but they still should